crm_lead_code/__manifest__.py

```diff
@@ -15,7 +15,7 @@
 #    GNU Affero General Public License for more details.
```

Comment:
For this file replace the whole comment block at the beginning with the shortened license and copyright lines.